crmd/remote_lrmd_ra.c

```diff
@@ -662,9 +685,14 @@ handle_remote_ra_stop(lrm_state_t * lrm_state, remote_ra_cmd_t * cmd)
     }
     ra_data->cmds = NULL;
     ra_data->recurring_cmds = NULL;
-    ra_data->cur_cmd = NULL;
 
-    report_remote_ra_result(cmd);
+    if (cmd) {
+        cmd->rc = PCMK_OCF_OK;
+        cmd->op_status = PCMK_LRM_OP_DONE;
+        ra_data->cur_cmd = NULL;
```

Comment:
cur_cmd is a pointer into cmds, so if we're freeing cmds unconditionally, we should set cur_cmd to NULL unconditionally too. What will happen if there is a pending command when the connection drops? Will the crmd get a timeout, or does it cancel the command somewhere?